.vscode/tasks.json

```diff
@@ -0,0 +1,16 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "cmake",
+			"label": "CMake: install",
+			"command": "install",
+			"problemMatcher": [],
+			"detail": "CMake template install task",
+			"group": {
+				"kind": "build",
+				"isDefault": true
+			}
+		}
+	]
+}
```

Comment:
Should this file be part of the `main` branch?

Reply:
Yes, it is helpful for those coding with vscode (us).

Reply:
More discussion here: https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control

Sharing tasks is a mentioned use case.